src/pages/AppConfigure.vue

```diff
@@ -1,748 +1,22 @@
 <script lang="ts" setup>
-import { ref } from '@vue/reactivity'
-import { computed } from '@vue/runtime-core'
-import { useRouter } from 'vue-router'
-import CircleProgress from 'vue3-circle-progress'
-import { useStore } from 'vuex'
+import { ref } from 'vue'
 
-import CloseIcon from '@/assets/iconography/close.svg'
-import DeleteIcon from '@/assets/iconography/delete.svg'
-import ConfigureAppAuth from '@/components/app-configure/AppAuth.vue'
-import ConfigureAppChainType from '@/components/app-configure/AppChain.vue'
-import ConfigureAppName from '@/components/app-configure/AppName.vue'
-import ConfigureAppRegion from '@/components/app-configure/AppRegion.vue'
-import ConfigureUserLimits from '@/components/app-configure/AppUserLimits.vue'
-import AppHeader from '@/components/AppHeader.vue'
-import ConfigureFooter from '@/components/ConfigureFooter.vue'
-import FullScreenLoader from '@/components/FullScreenLoader.vue'
-import VButton from '@/components/lib/VButton/VButton.vue'
-import VCard from '@/components/lib/VCard/VCard.vue'
-import VIconButton from '@/components/lib/VIconButton/VIconButton.vue'
-import VOverlay from '@/components/lib/VOverlay/VOverlay.vue'
-import VSeperator from '@/components/lib/VSeperator/VSeperator.vue'
-import VStack from '@/components/lib/VStack/VStack.vue'
-import VSwitch from '@/components/lib/VSwitch/VSwitch.vue'
-import VTextField from '@/components/lib/VTextField/VTextField.vue'
-import VTooltip from '@/components/lib/VTooltip/VTooltip.vue'
-import appConfigService from '@/services/app-config.service'
-import useArcanaAuth from '@/use/arcanaAuth'
-import cryptoUtils from '@/utils/cryptoUtils'
-import signerMakeTx from '@/utils/signerMakeTx'
+import ConfigureSidebar from '@/components/app-configure/ConfigureSidebar.vue'
+import constants from '@/utils/constants'
 
-const router = useRouter()
-const store = useStore()
-const arcanaAuth = useArcanaAuth()
+const configureTabs = constants.configureTabs
```

Comment:
We can skip these kinds of reassignments as they are not particularly helpful. Prefer using

```
import { type ConfigureTab, CONFIGURE_TABS } from '@/utils/constants' 
```

Reply:
Done